src/components/Profile/Profile.js

```diff
@@ -7,28 +7,20 @@ import Search from '../Search/Search';
 
 import './Profile.css';
 
-class Profile extends PureComponent {
```

Comment:
This component didn't require `PureComponent`, made into SFC.

Reply:
I read somewhere that PureComponent can't hurt but has a potential to help by avoiding unnecessary re-renders?  Something like that, been awhile.